clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -403,6 +405,16 @@ public void close() {
         client.disableWakeups();
         try {
             maybeAutoCommitOffsetsSync();
+
+            Node coordinator;
+            long endTimeMs = System.currentTimeMillis() + CLOSE_TIMEOUT_MS;
```

Comment:
We should probably use `time.milliseconds()`. Same for the other call in the loop below.